index.bs

```diff
@@ -837,11 +863,23 @@ url: https://w3c.github.io/permissions/#permissions-task-source; type: dfn;
 						
 						1. Set |representation|'s [=representation/MIME type=] to |mimeType|.
 
+						1. Let |isUnsanitized| be |false|.
```

Comment:
Does this step need to be done for each format (ie. inside the for each loop below)?

Reply:
Ah nice catch. Need to reset it at the end of the loop.